CLI.md

```diff
@@ -521,6 +521,13 @@ Run a GraphQL service that exposes a faucet where users can claim tokens. This g
   Default value: `8080`
 * `--amount <AMOUNT>` — The number of tokens to send to each new chain
 * `--limit-rate-until <LIMIT_RATE_UNTIL>` — The end timestamp: The faucet will rate-limit the token supply so it runs out of money no earlier than this
+* `--listener-skip-process-inbox` — Do not create blocks automatically to receive incoming messages. Instead, wait for an explicit mutation `processInbox`
```

Comment:
was the CI broken?

Reply:
Why would the CI be broken?

Reply:
Sorry, never mind. I just saw the change in `linera-client/src/client_options.rs` that explains it.